python/paddle/fluid/contrib/tests/README.md

```diff
@@ -0,0 +1,62 @@
+PaddlePaddle supports offline INT8 calibration to accelerate the inferene speed. In this document, we provide the instructions on how to enable INT8 calibration and show the results in both accuracy and performance.
+
+## 0. Prerequisite
+You need to build PaddlePaddle python package (e.g., paddlepaddle-0.0.0-cp27-cp27mu-linux_x86_64.whl) and install the package successfully.
+
+## 1. How to generate INT8 model
+You can refer to the unit test in test_calibration.py. Basically, there are three steps:
+* Construct calibration object. The required parameters are: 1) **program**: FP32 program; 2) **pretrained_model**: FP32 pretrained model; 3) **algo**: calibration algorithm; default is np.max, the alternative is KL (Kullback–Leibler divergence); 4) **exe**: executor; 5) **output**: INT8 model; 6) **feed_var_names**: feed dict; and 7) **fetch_list**: fetch targets. Here is the example:
+
+            calibrator = int8_utility.Calibrator( # Step 1
+                program=infer_program,
+                pretrained_model=model_path,
+                algo=algo,
+                exe=exe,
+                output=int8_model,
+                feed_var_names=feed_dict,
+                fetch_list=fetch_targets)
```

Comment:
move the parameters explanation into the codes.
```python
calibrator = int8_utility.Calibrator( # Step 1
                program=infer_program, # required, FP32 program
                pretrained_model=model_path, # required, FP32 pretrained model
                algo=algo,
                exe=exe,
                output=int8_model,
                feed_var_names=feed_dict,
                fetch_list=fetch_targets)
```

Reply:
Besides, you can use
```
```python
calibrator = int8_utility.Calibrator( # Step 1
   ...
```
to use the python format

Reply:
Done